programs/quarry-mine/src/account_validators.rs

```diff
@@ -255,17 +246,15 @@ impl<'info> Validate<'info> for UserStake<'info> {
 impl<'info> Validate<'info> for ExtractFees<'info> {
     fn validate(&self) -> ProgramResult {
         require!(!self.rewarder.is_paused, Paused);
-        assert_ata!(
+        assert_keys_eq!(
```

Comment:
Where is the corresponding owner check here?

Reply:
not needed since the account itself is checked